terraform/modules/forest_node/user-data.sh

```diff
@@ -31,8 +31,12 @@ echo "AllowUsers ${NEW_USER}" >> /etc/ssh/sshd_config
 
 systemctl restart sshd
 
+# Enable passwordless sudo for the new user. This allows the user to run sudo commands without being prompted for a password.
+echo "${NEW_USER} ALL=(ALL:ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/"${NEW_USER}"
+
 # Add new user to "sudo" and "docker" group so they can run docker commands and have general admin rights.
-usermod --append --groups sudo,docker "${NEW_USER}"
+usermod --append --groups docker "${NEW_USER}"
+
```

Comment:
Being in the `sudo` group isn't enough?

Reply:
Could you elaborate in more detail? Why is being in the `sudo` group no longer sufficient?

Reply:
Shouldn't the NewRelic agent be installed by `root` rather than the `chainsafe` user?